src/DoctrineORMModule/Form/Annotation/AnnotationBuilder.php

```diff
@@ -56,7 +56,8 @@ public function setEventManager(EventManagerInterface $events)
     {
         parent::setEventManager($events);
 
-        $this->getEventManager()->attach(new ElementAnnotationsListener($this->objectManager));
+        $annotationsListener = new ElementAnnotationsListener($this->objectManager);
+        $annotationsListener->attach($this->getEventManager());
```

Comment:
Can be inlined: `(new ElementAnnotationsListener($this->objectManager))->attach($this->getEventManager());`